tests/PSModuleTest.Tests.ps1

```diff
@@ -1,11 +1,16 @@
 ﻿
+BeforeAll {
+    . "$PSScriptRoot/../src/variables/private/UnitMap.ps1"
+    . "$PSScriptRoot/../src/functions/private/Format-UnitValue.ps1"
+    . "$PSScriptRoot/../src/functions/public/Format-TimeSpan.ps1"
+}
 
```

Comment:
We can remove this as its run by the test framework automatically after building the powershell module script.